src/main/java/org/prebid/server/spring/config/model/AccountConfigurationProperties.java

```diff
@@ -0,0 +1,58 @@
+package org.prebid.server.spring.config.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.settings.model.Account;
+import org.prebid.server.settings.model.AccountAnalyticsConfig;
+import org.prebid.server.settings.model.AccountGdprConfig;
+import org.prebid.server.settings.model.AccountStatus;
+import org.springframework.validation.annotation.Validated;
+
+@Validated
```

Comment:
Why this is required ?

Reply:
Good question. It's been copied along with other annotations that we put on Spring configuration properties POJOs but apparently there are no constraints imposed on any fields in this class so it's redundant. We may put sensible constraints in future and easily overlook adding `@Validated` annotation though so I'm hesitant about this one.